Model/Api/Tax.php

```diff
@@ -215,7 +225,8 @@ protected function getShippingDiscount($totalsInformation, $currencyCode, $shipp
             $this->quote,
             $this->quote->getShippingAddress()
         );
-        if ($shippingDiscountAmount >= DiscountHelper::MIN_NONZERO_VALUE) {
+
+        if ($shippingDiscountAmount >= DiscountHelper::MIN_NONZERO_VALUE && !$this->cartHelper->ignoreAdjustingShippingAmount($this->quote)) {
```

Comment:
Looks like this change is related to 3p party plugins support, Mirasvit credit, and Mirasvit rewards.
Is it relevant to cabinet's bug?

Reply:
It's not related to the Mirasvit credit, and Mirasvit rewards. I was able to replicate the issue a Magento (2.4.3) default store in my dev instance.

Reply:
Oh, my bad, I misread the code